Artifacts/MsSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,45 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @claim_Name NVARCHAR(MAX) = 'http://ed-fi.org/ods/identity/claims/domains/educationOrganizations';
+DECLARE @educationOrganizationsResourceClaimId INT;
+DECLARE @claimSet_Id INT;
+DECLARE @claimSet_Name NVARCHAR(MAX) = 'Bootstrap Descriptors and EdOrgs';
+DECLARE @msg NVARCHAR(MAX);
+IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = @claim_Name)    
+BEGIN
+	SET @msg = 'ClaimName does not exist';
+    THROW 50000, @msg, 1
+END
+
+IF NOT EXISTS (SELECT 1 FROM dbo.ClaimSets WHERE ClaimSetName = @claimSet_Name)    
+BEGIN
+   SET @msg = 'ClaimSetName does not exist';
+   THROW 50000, @msg, 1
+END
+
+IF  EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = @claim_Name)    
+BEGIN
+	SELECT	@educationOrganizationsResourceClaimId = ResourceClaimId
+	FROM	dbo.ResourceClaims rc
+	WHERE	rc.ClaimName = @claim_Name
+END
+
+IF  EXISTS (SELECT 1 FROM dbo.ClaimSets WHERE ClaimSetName = @claimSet_Name)    
```

Comment:
As above, this check is also redundant with the one above which throws an error. Remove this one.